edx_solutions_api_integration/courses/views.py

```diff
@@ -3,58 +3,52 @@
 import itertools
 import logging
 import sys
-from StringIO import StringIO
+import warnings
+
 from collections import OrderedDict
 from datetime import timedelta
+from StringIO import StringIO
 
-from completion.models import BlockCompletion
-from completion_aggregator.models import Aggregator
-from courseware.courses import (
-    get_course_about_section,
-    get_course_info_section,
-    get_course_info_section_module,
-)
-from courseware.models import StudentModule
-from courseware.views.views import get_static_tab_fragment
 from django.conf import settings
 from django.contrib.auth.models import Group, User
 from django.core.cache import cache
 from django.core.exceptions import ObjectDoesNotExist
 from django.db.models import Count, F, Max, Min, Q
 from django.http import Http404
 from django.utils.translation import ugettext_lazy as _
-from opaque_keys.edx.keys import UsageKey
 
+from lxml import etree
 from requests.exceptions import ConnectionError
-
 from rest_framework import status
 from rest_framework.response import Response
 
-from courseware.courses import get_course_about_section, get_course_info_section, get_course_info_section_module
+from completion.models import BlockCompletion
+from completion_aggregator.models import Aggregator
+from course_metadata.models import CourseAggregatedMetaData
+from courseware.courses import (
+    get_course_about_section,
+    get_course_info_section,
+    get_course_info_section_module,
+)
 from courseware.models import StudentModule
 from courseware.views.views import get_static_tab_fragment
-from mobile_api.course_info.views import apply_wrappers_to_content
-from openedx.core.lib.xblock_utils import get_course_update_items
-from openedx.core.lib.courses import course_image_url
-from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
-from openedx.core.djangoapps.content.course_structures.api.v0.errors import CourseStructureNotAvailableError
 from django_comment_common.models import FORUM_ROLE_MODERATOR
+from gradebook.models import StudentGradebook
 from instructor.access import revoke_access, update_forum_role
 from lms.djangoapps.course_api.blocks.api import get_blocks
 from lms.lib.comment_client.thread import get_course_thread_stats
 from lms.lib.comment_client.user import get_course_social_stats
-from lms.lib.comment_client.utils import CommentClientMaintenanceError, CommentClientRequestError
-from lxml import etree
+from lms.lib.comment_client.utils import (
+    CommentClientMaintenanceError,
+    CommentClientRequestError
+)        import warnings
```

Comment:
@jcdyer, this line is missing a newline character before the `import` statement which has to be on a separate line.